osu.Game/Graphics/UserInterfaceV2/RoundedButton.cs

```diff
@@ -26,19 +36,67 @@ public override float Height
             }
         }
 
+        public override Color4 BackgroundColour
+        {
+            get => base.BackgroundColour;
+            set
+            {
+                base.BackgroundColour = value;
+                triangleGradientSecondColour = BackgroundColour.Lighten(0.2f);
+                updateColours();
+            }
+        }
+
         [BackgroundDependencyLoader(true)]
-        private void load(OsuColour colours)
+        private void load(OverlayColourProvider? overlayColourProvider, OsuColour colours)
         {
             // According to flyte, buttons are supposed to have explicit colours for now.
             // Not sure this is the correct direction, but we haven't decided on an `OverlayColourProvider` stand-in yet.
             // This is a better default. See `SettingsButton` for an override which uses `OverlayColourProvider`.
-            DefaultBackgroundColour = colours.Blue3;
+            DefaultBackgroundColour = overlayColourProvider?.Colour3 ?? colours.Blue3;
+            triangleGradientSecondColour ??= overlayColourProvider?.Colour1 ?? colours.Blue3.Lighten(0.2f);
         }
 
         protected override void LoadComplete()
         {
             base.LoadComplete();
+
             updateCornerRadius();
+
+            Add(triangles = new TrianglesV2
+            {
+                Thickness = 0.02f,
+                SpawnRatio = 0.6f,
```

Comment:
Not sure whether that was intended, but this spawn ratio makes it so that the smaller buttons (i.e. key binding row buttons, test browser input source selector buttons) have barely any triangles, if any at all. Maybe fine to keep it subdued, but just saying as I'm seeing it.

Reply:
Good catch. I've adjusted the spawn ratio to only be based on width now (using one of the test scene's button's `DrawHeight` to update the ratio value).